canonical/canonical.com
# Create /solutions/telco/5g-core webpage

## Done

- Create new webpage for /solutions/telco/5g-core
- Create dynamic form for contact CTA
- Added /macros/quote_wrapper.jinja from Vanilla, the file is currently being pasted manually but will be able to import from Vanilla directly once it is released
- Add secondary navigation for Telco bubble

## QA

- Go to https://canonical-com-1380.demos.haus/solutions/telco/5g-core
- Check that content matches [copy doc](https://docs.google.com/document/d/1h5A07aTGTZl5VfoiX2UwIpb-bHtFr7gBVUejN-uAf-Q/edit) and design matches [Figma](https://www.figma.com/design/brgzRViEncgKKD4KzVnyXy/24.10-Canonical.com%2Fsolutions?node-id=458-7435&node-type=instance&m=dev)
- Check that contact us modal form works and submits to Marketo successfully
- Check with medium and small screens

## Issue / Card

Fixes [WD-14135](https://warthogs.atlassian.net/browse/WD-14135)

## Screenshots

[if relevant, include a screenshot]


[WD-14135]: https://warthogs.atlassian.net/browse/WD-14135?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
